### PR TITLE
BEER tutorial

### LINKED ISSUE #151: Add BEER tutorial
Add a section to the tutorial explaining the BEER metric and how to use it.

### PR DESCRIPTION
**Pull request by EyalLavi**
Fixes #151

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -250,3 +250,109 @@ Word Error Rate variants
...
+
+Consider for example this sentence in the reference transcript: 'The European Union headquarters'. If engine A returns 'The European onion headquarters' and engine B returns 'The European Union headache', the Word Error Rate would be similar for both engines since in both cases one word was transcribed inaccurately. But engine B should be 'rewarded' for preserving the phrase 'European Union'. The BEER is the metric that takes such considerations into account. 
+
+Another use for this metric is compensating for distortions of WER that are caused by normalization rules. For example, you may convert both reference and hypothesis transcripts to lower case or remove punctuation marks so that they don't affect the WER. In this case, the distinction between 'Theresa May' and 'Therese may' is lost. But you can instruct BenchmarkSTT to score higher the engine that produced 'Theresa May'.
```
Comment (alexandre-rouxel) [inline docs/tutorial.rst:264]: 'Theresa may'

File: docs/tutorial.rst [hunk 2]
```diff
@@ -250,3 +250,109 @@ Word Error Rate variants
...
+
+The file has this structure: 
+
+   ``{ "entity1" : weight1, "entity2" : weight2, "entity3" : weight2 .. }``
+
+Let's create an example list. Save the below list as file ``entities.json``::
+
+   {"Theresa May" : 0.5, "Abigail" : 0.5, "EU": 0.75, "Griffin" : 0.5, "I" : 0.25}
+
+We'll also tell BenchmarkSTT to normalize the reference and hypothesis file by lowercasing both. We do this in the ``config.conf`` file: ::
```
Comment (aro-max) [inline docs/tutorial.rst]: We'll also tell BenchmarkSTT to normalize the reference and hypothesis file but without lowercasing both. We do this in the ``config.conf`` file: ::

File: docs/tutorial.rst (hunk 1 above)
Comment (aro-max) [inline docs/tutorial.rst]: the distinction between 'Theresa May' and 'Theresa may' is lost.

**Review by aro-max — CHANGES_REQUESTED** [review]

**Comment by aro-max** [comment]
I merged a new pull request with minor changes on the BEER tutorial.

**Pull request closed without merging**